src/content/sellers-guide/index.mdx

```diff
@@ -12,12 +12,16 @@ import ContentListItem from "@components/ContentListItem.astro"
 We are currently in the process of developing the Sellers' guide, which will help you manage your contracts, get on OASIS+ and find resources to get the most out of your experience. 
 Below is a preview of the guide topics. 
 
-<Alert heading="Upcoming dates and events">
-- Current solicitation status: Closed on October, 20, 2023. 
-- Next proposal submission opportunity: No later than FY2025; they will be continuously open from that point forward.
-
-Visit the [OASIS+ Interact Community for updates.](https://buy.gsa.gov/interact/community/196/activity-feed)
-</Alert>
+   <USWSDAlert heading="Updates and announcements">
```

Comment:
You probably need to change USWDSAlert back to Alert as the import statement give alias name as Alert. Or you can change the top import statement from "import Alert from "@components/USWDSAlert.astro;" to "import USWDSAlert from "@components/USWDSAlert.astro;"